backend/routes/health.py

```
@@ -0,0 +1,42 @@
+import time
+
+from fastapi import APIRouter, Depends
+
+from ..core.config import AppConfig
+from ..dependencies import get_app_state
+
+router = APIRouter()
+
+
+@router.get("/")
+async def root(state: dict = Depends(get_app_state)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)